renovate.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base"
+  ],
+  "reviewers": ["team:data-platform-mysql"],
```

Comment:
I couldn't test if this will pull the team members or just assign the team on my repo, so it may need to be tweaked.

Reply:
Most probably this should be `canonical/data-platform-mysql`  or `@canonical/data-platform-mysql`

Reply:
> Most probably this should be `canonical/data-platform-mysql` or `@canonical/data-platform-mysql`

[Docs](https://docs.renovatebot.com/configuration-options/#reviewers) say to skip it

Reply:
Roger.

Reply:
This will just assign the team.